src/Build/Graph/ProjectInterpretation.cs

```diff
@@ -191,8 +191,6 @@ public void ReparentInnerBuilds(Dictionary<ConfigurationMetadata, ParsedProject>
 
                             if (outerBuildReferencingProject.ProjectReferences.Contains(innerBuild))
                             {
-                                graphBuilder.Edges.TryGetEdge((outerBuildReferencingProject, innerBuild), out var existingEdge);
```

Comment:
Not entirely sure how this is related, why was this needed?

Reply:
It's unused :). The lines below just call `graphBuilder.Edges[]` directly so `existingEdge` isn't used at all.